client/blocks/importer/wordpress/upgrade-plan/constants.tsx

```diff
@@ -49,23 +49,26 @@ export const UpgradePlanHostingTestimonials = [
 	},
 ];
 
-export const defaultHostingDetails = [
-	{
-		id: 'reduced-error-rate',
-		title: translate( 'Reduced error rate' ),
-		description: translate( '16% fewer errors' ),
-		icon: shield,
-	},
-	{
-		id: 'increased-speed',
-		title: translate( 'Increased speed' ),
-		description: translate( '30% faster' ),
+export const defaultHostingDetails: {
+	[ key: string ]: {
+		title: string;
+		description: string | ReactNode;
+		icon: JSX.Element;
+	};
+} = {
+	'higher-speed': {
+		title: translate( 'Higher speed' ),
+		description: translate( '30% faster.' ) + '*',
 		icon: trendingUp,
 	},
-	{
-		id: 'higher-availability',
+	'faster-response': {
+		title: translate( 'Faster response' ),
+		description: translate( '98% of sites on WordPress.com have a fast response.' ),
```

Comment:
The base of this calc is a round from 97.8% (April 2024 data - https://cwvtech.report/)